### PR TITLE
update readme, better project template

### PR DESCRIPTION
**Pull request by sergpolly**
addressing https://github.com/mirnylab/distiller-nf/issues/64

 - there is updated README
 - test project with chunk size 6000 moved to test
 - new "better" project.yml is in the main folder

### CODE REVIEW

File: project.yml [hunk 1]
```diff
@@ -1,86 +1,137 @@
...
-#                - sra:SRR2601842
-#                 - sra:SRR2601843?start=0&end=10000
-            lane2:
-#                 Files can also be provided as http/ftp/s3 links
-                - https://github.com/mirnylab/distiller-test-data/raw/master/fastq/MATalpha_R1/lane2/SRR2601843_1.fastq.gz
-                - https://github.com/mirnylab/distiller-test-data/raw/master/fastq/MATalpha_R1/lane2/SRR2601843_2.fastq.gz
-#                - ./test/fastq/MATalpha_R1/lane2/SRR2601843_1.fastq.gz
-#                - ./test/fastq/MATalpha_R1/lane2/SRR2601843_2.fastq.gz
-        MATalpha_R2:
+                - /path/to/your/fastq/from/library1/library1_R1.fastq.gz
```
Comment (golobor) [inline project.yml]: _R1/_R2 notation for the two sides of the library are a bit confusing (looks like replicate1/replicate2). However, that's the Illumina's notation, correct?

Comment (Phlya) [reply project.yml]: You can really get a wide variety of naming schemes from different facilities, I think this doesn't matter...

**Comment by sergpolly** [comment]
agree! if forgot that those are sides - I used to think of them as left/right  - never mind...

`_side1, _side2` - seems super understandable!

File: project.yml (hunk 1 above)
Comment (sergpolly) [reply project.yml]: sides seems great! _side1/2

**Pull request merged**